R/openxlsx2.R

```diff
@@ -6,7 +6,7 @@
 #' Based on a powerful XML library and focusing on modern programming flows in pipes
 #' or chains, `openxlsx2` allows to break many new ground.
 #'
-#' @name openxlsx2
+#' @name openxlsx2-package
```

Comment:
Maybe could rename the file too to make it easier to navigate to in RStudio.

Reply:
I don't have a strong opinion on this. Personally I just like to think of my projects independently of IDEs, but we can think about renaming files.